sklearn/datasets/_samples_generator.py

```diff
@@ -855,6 +855,18 @@ def make_moons(n_samples=100, *, shuffle=True, noise=None, random_state=None):
     return X, y
 
 
+@validate_params(
+    {
+        "n_samples": [Interval(Integral, 1, None, closed="left"), "array-like"],
+        "n_features": [Interval(Integral, 1, None, closed="left")],
+        "centers": [Interval(Integral, 1, None, closed="left"), "array-like", None],
```

Comment:
Since center can be an array-like, please replace `ndarray` by `array-like` in the docstring.

Reply:
Thanks for the feedback, I updated the docstring.